src/style.css

```diff
@@ -1,3 +1,58 @@
-body {
+* {
   margin: 0;
 }
+
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  width: 1200px;
```

Comment:
Use max and min width to keep your header more flexible

Reply:
done